### PR TITLE
Labels annotations flag icon

### PR DESCRIPTION
**Pull request by lynnerang**

### CODE REVIEW

File: src/views/AccessibilityBase.svelte [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
   export let items = null;
   export let viewContext = 'a11y-keyboard';
+
```
Comment (gblakeman) [inline src/views/AccessibilityBase.svelte:7]: Any reason for this extra line?

File: src/views/ItemHeader.svelte [hunk 2]
```diff
@@ -107,7 +108,9 @@
       />
     </span>
     <span class="text">
+      {#if showErrorIcon}
       <span class="error-flag">&#9873;&nbsp;</span>
```
Comment (gblakeman) [inline src/views/ItemHeader.svelte:112]: Might be a matter of preference, but I like to indent the markup inside of `if` statements of a Svelte view.

File: src/views/ItemsList.svelte [hunk 3]
```diff
@@ -30,6 +30,20 @@
     return itemIsOpen;
   };
 
+  const isMissingData = (item) => {
+    if (type.includes('labels')) {
+      const { labels, role } = item;
+      return (
```
Comment (gblakeman) [inline src/views/ItemsList.svelte:36]: This is such a dense conditional, we should probs have a comment explaining what it does.

File: src/additionalTypes.d.ts [hunk 4]
```diff
@@ -4,6 +4,10 @@ declare global {
 
   type PluginKeystopKeys = 'arrows-left-right' | 'arrows-up-down' | 'enter' | 'escape' | 'space';
 
+  type PluginLabelRoles = 'image' | 'image-decorative' | 'button' | 'checkbox' | 'link' | 'menuitem' | 'menuitemcheckbox' | 'menuitemradio' | 'options' | 'progressbar' | 'searchbox' | 'radio' | 'slider' | 'switch' | 'tab' | 'tabpanel' | 'textbox' | 'combobox' | 'listbox' | 'menu' | 'radiogroup' | 'tablist';
+
+  // type PluginLabelRoles = 'image' | 'button' |
```
Comment (gblakeman) [inline src/additionalTypes.d.ts:9]: Seems like this line is no longer needed?

**Review by gblakeman — CHANGES_REQUESTED** [review]
This looks amazing for your first PR 🌈🦄

I just flagged a few small things.

And when we chat, I want to figure out why your `package-lock.json` file is so different from mine? It might just be a version discrepancy in `npm` or something like that…

File: src/App.ts [hunk 5]
```diff
@@ -1732,6 +1732,54 @@ export default class App {
     return null;
   }
 
+  /**
```
[the remark this answers is not in the document]
Comment (lynnerang) [reply src/App.ts:1735]: 👍  just added updated doc, will make sure to add WIP for the others while they're in flux

File: src/views/ItemHeader.svelte (hunk 2 above)
Comment (lynnerang) [reply src/views/ItemHeader.svelte:112]: I dig it, adding an indent now, thanks!

File: src/views/ItemsList.svelte (hunk 3 above)
Comment (lynnerang) [reply src/views/ItemsList.svelte:36]: cool, added!

File: src/views/AccessibilityBase.svelte (hunk 1 above)
Comment (lynnerang) [reply src/views/AccessibilityBase.svelte:7]: No reason, just pulled it, thanks!

File: src/additionalTypes.d.ts (hunk 4 above)
Comment (lynnerang) [reply src/additionalTypes.d.ts:9]: Deleted, thanks!

**Review by gblakeman — APPROVED** [review]
🎉👊

**Pull request merged**